src/main/scala/com/fulcrumgenomics/sv/BreakpointPileup.scala

```diff
@@ -32,12 +34,15 @@ case class BreakpointPileup(id: String,
                             right_strand: Char,
                             split_reads: Int,
                             read_pairs: Int,
-                            total: Int
+                            total: Int,
+                            left_target: Option[String] = None,
+                            right_target: Option[String] = None
                            ) extends Metric {
 
   override def toString(): String = f"$id|$left_contig:$left_pos($left_strand)/" +
     f"$right_contig:$right_pos($right_strand)|" +
-    f"$split_reads,$read_pairs,$total"
+    f"$split_reads,$read_pairs,$total" +
+    f"${left_target.getOrElse("")},${right_target.getOrElse("")}"
```

Comment:
CSVs within CSVs!
Seriously though - I think you a) need something between $total and ${left...}, and b) might want to to `left_target.map(_.replace(',' ';')` or similar so it's clear what's what.  Otherwise how do I tell what are the left and right targets?